documentation/docs/references/client/classes/B1T6.md

```diff
@@ -0,0 +1,62 @@
+# Class: B1T6
```

Comment:
Since this all is autogenerated, we could look into it to generate some basic docusaurus metadata like keywords.